packages/amplify-provider-awscloudformation/src/configuration-manager.js

```diff
@@ -19,11 +24,26 @@ async function init(context) {
     return context;
   }
   normalizeInputParams(context);
-  context.exeInfo.awsConfigInfo = {
-    configLevel: 'project',
-    config: defaultAWSConfig,
-  };
-  await newUserCheck(context);
+
+  let meta = undefined;
+  try {
+    meta = stateManager.getMeta(undefined, { throwIfNotExist: false });
```

Comment:
no need for try catch, hence the `throwIfNotExists` parameter ;-)

Reply:
it is needed because on sandbox pull it's possible to hit this point before the folder is a valid amplify project which causes getMeta to fail because it doesn't know what the project dir is and that happens before the file existence check